### ISSUE TITLE
input type=color - last example does not work in two browsers

### ISSUE THREAD

**Issue by joyously** [mdn/browser-compat-data#20077]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/color#result

### What information was incorrect, unhelpful, or incomplete?

The example explains two different events (input and change), but only the change event actually changes the color. The input event does not do what it says it should.

### What did you expect to see?

I expected to see the paragraph color change as I chose different colors in the color picker, as the page says it should.

### Do you have anything more you want to share?

I am on Ubuntu 22.04, and tried in Firefox 114 and Falkon 3.2.0. It looks like both are using the system color picker, which is likely the problem. 
But the explanation should account for having only one event at the end of selection instead of implying that both events are always triggered.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/input/color`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/input/color/index.md
* Last commit: https://github.com/mdn/content/commit/1995a9dbe407d9ee41c3b1b50f29e020a85e456a
* Document last modified: 2023-03-31T00:59:22.000Z

</details>

**Comment by wbamberg** [comment]
@joyously, do you have Chrome on Linux, and if so, so you see the same thing there?

**Comment by joyously** [comment]
No, I don't install Chrome due to its terms of service.

**Comment by caugner** [comment]
Closing as stale for now.

I have tested the example on macOS 26.3.1, and it fully worked in Chrome 146, Firefox 148, Safari 26.3.1.

@joyously If you still experience the issue, please file a [new Firefox bug in the `DOM: Forms` component](https://bugzilla.mozilla.org/enter_bug.cgi?product=Core&component=DOM%3A%20Forms). 🙏

**Comment by joyously** [comment]
@caugner I thought this repo was about compatibility data issues. I just tested again on Ubuntu 24.04 with Firefox 148.0.2, and it does exactly the same thing as before. The `input` event is not triggered (I'm guessing the system color picker does not propagate it back), and the compatibility data does not mention this possibility. The implementation notes do point to a bug about implementing `datalist`, but that is working, so _that_ is stale...

**Comment by caugner** [comment]
Thanks for confirming, and sorry to hear this is still an issue.

In order to track the resolution of this issue, we need a Firefox bug that we can reference in the BCD data, so we can update it when this gets resolved. 🙏

**Comment by joyously** [comment]
The compatibility table doesn't have an entry for events. Perhaps that could be added, so that Firefox could be appropriately labeled as different from the others.

**Comment by caugner** [comment]
> The compatibility table doesn't have an entry for events. Perhaps that could be added, so that Firefox could be appropriately labeled as different from the others.

Good point, though note that `HTMLInputElement` inherits [the `change` event from `HTMLElement`](https://github.com/mdn/browser-compat-data/blob/b80edbd4797faac84cfabc92fe3602ad1e923f61/api/HTMLElement.json#L575) and [inherits the `input` event from `Element`](https://github.com/mdn/browser-compat-data/blob/b80edbd4797faac84cfabc92fe3602ad1e923f61/api/Element.json#L6153). Although [the DOM event guidelines](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/api.md#dom-events-eventname_event) don't mention this, we don't currently capture inherited events.

@ddbeck (cc @Elchi3) Would it make sense to create a subfeature for an inherited event if its support differs? (Assuming we have a Firefox bug for reference.)

**Comment by ddbeck** [comment]
> [@ddbeck](https://github.com/ddbeck) (cc [@Elchi3](https://github.com/Elchi3)) Would it make sense to create a subfeature for an inherited event if its support differs? (Assuming we have a Firefox bug for reference.)

I'm generally averse to creating a subfeature for to capture a single implementer's bug, especially when we have a situation where it's affecting a specific OS. I would suggest adding a note to the `input_event` key (even though it'll be a bit buried there), linking to the relevant bug. It's not great, but the scope of this is so narrow, I think it's best trade off for maintainability.

We would still need a bug filed against Firefox though. I don't want BCD to be the sole tracker of a bug.
